nuxeo-mqueues-core/src/main/java/org/nuxeo/ecm/platform/importer/mqueues/mqueues/chronicle/ChronicleMQueue.java

```diff
@@ -177,27 +217,41 @@ private ChronicleMQueue(File basePath, int size) {
         }
         this.name = basePath.getName();
         this.basePath = basePath;
+
+        if (retentionDuration != null) {
+            retentionNbCycles = Integer.valueOf(retentionDuration.substring(0, retentionDuration.length() - 1));
+        }
+
+        final RollCycle rollCycle = getRollCycle(retentionDuration);
```

Comment:
No need for `final`, it's assigned only once so is _effectively final_ which is enough for lambda closures.